Deprecated the old 3-args form of `docker import`

vdemeester opened the pull request:
It's been deprecated since November 2013 and v0.6.7 (see https://github.com/docker/docker/commit/ef57752bce5c333d0e2b8352a84d071f21cce132). Removing the cli side of it 🐮

`docker import myimage.tgz myimage mytag` is no more :angel:.

/cc @thaJeztah @icecrime @stevvooe 

🐸

Signed-off-by: Vincent Demeester vincent@sbr.pm

thaJeztah commented:
@vdemeester looks like you need to remove an import;

```
16:05:37 # github.com/docker/docker/api/client
16:05:37 api/client/import.go:4: imported and not used: "fat"
```

vdemeester committed:
Deprecated the old 3-args form of `docker import`

It's been deprecated since November 2013 and v0.6.7. Removing the cli
side of it.

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

thaJeztah commented:
LGTM

vdemeester commented:
@thaJeztah yeah saw that :stuck_out_tongue_closed_eyes: but the import is not `fat` but `fmt` :joy:

thaJeztah commented:
@vdemeester my autocorrect disagreed, :joy:

runcom commented:
LGTM

stevvooe commented on docs/deprecated.md:
```diff
@@ -14,6 +14,13 @@ weight=80
+
+**Removed In Release: [v1.12.0](https://github.com/docker/docker/releases/tag/v1.12.0)**
+
+The `docker import` command format 'file|URL|- [REPOSITORY [TAG]]' is deprecated since November 2013. It's no more supported.
```
> It's no more supported.

Indeed!
